docs/src/modules/components/AppSearch.js

```diff
@@ -447,7 +447,7 @@ export default function AppSearch(props) {
             initialQuery={initialQuery}
             appId="TZGZ85B9TB"
             apiKey="8177dfb3e2be72b241ffb8c5abafa899"
-            indexName="material-ui"
+            indexName="v5_material-ui"
```

Comment:
Will we need to bump the monorepo and redeploy old `mui-x` docs to benefit from this? 🤔 
That could prove... Difficult. 😆 

WDYT about mui-x v5, v6 and v7?
Do I understand correctly, that everything using the old or latest `@mui/monorepo` will use the latest index and any branch, which merges a monorepo with this `indexName` will use the index for `material-ui` v5? 🤔

Reply:
About v5, v6, v7, we can move the dicsussion in https://github.com/mui/mui-x/issues/14855#issuecomment-2413181153

About bumping monorepo, we can create branches mui-x_v5.x_monorepo which is the latest dependency + this commit. And move this Algolia config in some env settings

Reply:
Yes, having this index configurable with environment variables sounds way more flexible than relying on a specific monorepo version to be merged. 🙈 👍